### PR TITLE
Outputter support for runners

### PR DESCRIPTION
**Pull request by cachedout**

### CODE REVIEW

File: salt/client/mixins.py [hunk 1]
```diff
@@ -433,7 +433,7 @@ def print_async_event(self, suffix, event):
         # more general, since this will get *really* messy as
         # people use more events that don't quite fit into this mold
         if suffix == 'ret':  # for "ret" just print out return
-            salt.output.display_output(event['return'], None, self.opts)
+            salt.output.display_output(event['return'], self.opts.get('out', None), self.opts)
```
Comment (kiorky) [inline salt/client/mixins.py]: uhm if i remember well we (with @jacksont)j disabled this as for runners, we expect raw results directly from the runner itself to be ouputted to stdout, if i understand well what you are trying to do there, im not ok :D

Comment (cachedout) [inline salt/client/mixins.py]: If an output option is not explicitely specified on the command-line, the behaviour is exactly the same as it was before.

Comment (kiorky) [inline salt/client/mixins.py]: yep i confirm it is ok. (tested in our install)

**Comment by jacksontj** [comment]
Looks good :)

**Pull request merged**